portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/webcontentdisplay/WebContentDisplay.testcase

```diff
@@ -502,6 +502,50 @@ Welcome to Liferay Community Edition Portal 7.4.0 CE GA1''';
 			webContentTextFieldLabel = "Text Box");
 	}
 
+	@description = "This is a test for LPS-83798. Validate File system can't access using Journal/DDM templates."
+	@priority = "5"
+	test AssertContentNotDisplayWhenUseJournalTemplates {
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+		NavItem.gotoStructures();
+
+		WebContentStructures.addCP(structureName = "WC Structure Name");
+
+		DataEngine.addField(
+			fieldFieldLabel = "Text",
+			fieldName = "Text");
+
+		WebContentStructures.saveCP(structureName = "WC Structure Name");
+
+		NavItem.gotoTemplates();
+
+		var templateScript = '''
+${httpUtil.URLtoString(request.getAttribute("CTX").getResource(".").toURI().create("file:///C:/boot.ini").toURL())}
```

Comment:
@chaowang-123 this will only work on a Windows file system.  We can use a conditional here (like [this one](https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/macros/fragment/FragmentsAPI.macro#L10-L14)) to use Windows if true, else make a templateScript with the Linux/Mac format.